.plzconfig

```diff
@@ -47,7 +47,7 @@ targetlevel = 7
 
 [proto]
 protoctool = //third_party/proto:protoc
-pythonpackage = third_party.python.google.protobuf
+pythonpackage = google.protobuf
```

Comment:
It might be possible to totally remove this (which would be a nice cleanup). I might have a little look after it's in.